File: Data Structures & Algorithms/remove-element/submission-0.cpp
class Solution {
public:
    int removeElement(vector<int>& nums, int val) {
        int k = 0;
        for (auto itr = nums.begin(); itr != nums.end();) {
            if (val == *itr) nums.erase(itr);
            else {
                ++itr;
                ++k;
            }
        }

        return k;
    }
};

File: Data Structures & Algorithms/jump-game/submission-0.py
class Solution:
    def canJump(self, nums: List[int]) -> bool:
        if not nums or len(nums) == 1:
            return True
        jump = nums[0]
        for i in range(jump):
            if self.canJump(nums[i+1:]):
                return True

        return False